ytfzf

```diff
@@ -159,10 +159,11 @@ create_subs () {
     # check how many subscriptions there are in the file
     sublength=$( jq '. | length' < "$yt_sub_import_file" )
 
+    subscriptions_json="$(cat "$yt_sub_import_file")"
```

Comment:
the way it was before is better. Since we are calling lesser subshells. Storing in a variable means we would need to pipe it, we would be invoking another subshell due to the pipe